backend/repositories/MatchRepository.ts

```diff
@@ -92,13 +94,13 @@ const getElapsedSecondsSinceLatestMatch = async (): Promise<number> => {
   * @param {Array<number>} matchDescription.team2 Array of 1 or 2 elements containing IDs of players from team2.
   * @param {boolean} matchDescription.team1Won True if team1 won, false if team2 won.
   */
-export const recordMatch = async (matchDescription): Promise<Match> => {
+export const recordMatch = async (gameName: string, matchDescription): Promise<Match> => {
```

Comment:
i'd add an interface/type for matchDescription, and also add check isMatchDescription, like did in the friday's PR

Reply:
it is already added here https://github.com/salsita/foosball-rating/pull/129/commits/69717c292c04a2fac323ec8cb31289dfdbdbf8b8 and I will adjust it as in the friday's PR. Is it ok with you?

Reply:
fine with me!